src/middleware/submissionHandler.js

```diff
@@ -383,6 +383,56 @@ module.exports = function(router, resourceName, resourceId) {
       done();
     };
 
+    /**
+     * Set parenthood of child forms if not already done
+     *
+     * @param req
+     * @param res
+     * @param done
+     */
+    var setChildFormParenthood = function(req, res, done) {
+      if (req.method !== 'POST' && req.method !== 'PUT') {
+        return done();
+      }
+
+      _.each(req.flattenedComponents, function(component, key) {
```

Comment:
Thanks Travis for the speedy and detailed guidance.  Will investigate and try to reorganize the code tomorrow.  I was assuming this needed to be done when the parent was being POSTed or PUT so that the parent id, res.resource.item._id, would be available.